Input Notebook:  /data/aglinska/BC-fMRI-AE/Notebooks/004-ABIDE-I-and-II-compcorr-filter.ipynb
Output Notebook: /data/aglinska/BC-fMRI-AE/Assets/data_preprocessed_f3_notebooks/03_preprocess_1864.ipynb
Generating grammar tables from /data/aglinska/anaconda3/lib/python3.8/site-packages/blib2to3/Grammar.txt
Writing grammar tables to /data/aglinska/.cache/black/21.8b0/Grammar3.8.5.final.0.pickle
Writing failed: [Errno 2] No such file or directory: '/data/aglinska/.cache/black/21.8b0/tmpa9qonb5s'
Generating grammar tables from /data/aglinska/anaconda3/lib/python3.8/site-packages/blib2to3/PatternGrammar.txt
Writing grammar tables to /data/aglinska/.cache/black/21.8b0/PatternGrammar3.8.5.final.0.pickle
Writing failed: [Errno 2] No such file or directory: '/data/aglinska/.cache/black/21.8b0/tmpa5mko775'
Executing:   0%|          | 0/73 [00:00<?, ?cell/s]Executing notebook with kernel: python3
Executing:   1%|▏         | 1/73 [00:04<05:27,  4.55s/cell]Executing:   4%|▍         | 3/73 [00:21<06:40,  5.71s/cell]Executing:   5%|▌         | 4/73 [00:21<04:38,  4.03s/cell]Executing:   8%|▊         | 6/73 [00:21<03:10,  2.84s/cell]Executing:  11%|█         | 8/73 [00:21<02:10,  2.01s/cell]Executing:  14%|█▎        | 10/73 [00:21<01:29,  1.43s/cell]Executing:  16%|█▋        | 12/73 [00:22<01:02,  1.02s/cell]Executing:  16%|█▋        | 12/73 [00:39<01:02,  1.02s/cell]Executing:  18%|█▊        | 13/73 [00:52<09:45,  9.76s/cell]Executing:  21%|██        | 15/73 [00:52<06:37,  6.85s/cell]Executing:  23%|██▎       | 17/73 [00:52<04:29,  4.81s/cell]Executing:  26%|██▌       | 19/73 [00:52<03:02,  3.39s/cell]Executing:  29%|██▉       | 21/73 [00:52<02:04,  2.39s/cell]Executing:  32%|███▏      | 23/73 [00:52<01:24,  1.69s/cell]Executing:  34%|███▍      | 25/73 [00:53<01:01,  1.27s/cell]Executing:  37%|███▋      | 27/73 [00:53<00:41,  1.10cell/s]Executing:  40%|███▉      | 29/73 [00:53<00:28,  1.52cell/s]Executing:  42%|████▏     | 31/73 [00:53<00:20,  2.09cell/s]Executing:  45%|████▌     | 33/73 [00:54<00:14,  2.68cell/s]Executing:  48%|████▊     | 35/73 [00:54<00:10,  3.53cell/s]Executing:  48%|████▊     | 35/73 [01:10<00:10,  3.53cell/s]Executing:  49%|████▉     | 36/73 [02:13<14:44, 23.91s/cell]Kernel died while waiting for execute reply.
Executing:  51%|█████     | 37/73 [06:13<53:14, 88.73s/cell]Executing:  51%|█████     | 37/73 [06:14<06:03, 10.11s/cell]
Traceback (most recent call last):
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 816, in async_execute_cell
    exec_reply = await self.task_poll_for_reply
asyncio.exceptions.CancelledError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/papermill/cli.py", line 250, in papermill
    execute_notebook(
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/papermill/execute.py", line 107, in execute_notebook
    nb = papermill_engines.execute_notebook_with_engine(
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/papermill/engines.py", line 49, in execute_notebook_with_engine
    return self.get_engine(engine_name).execute_notebook(nb, kernel_name, **kwargs)
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/papermill/engines.py", line 343, in execute_notebook
    cls.execute_managed_notebook(nb_man, kernel_name, log_output=log_output, **kwargs)
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/papermill/engines.py", line 402, in execute_managed_notebook
    return PapermillNotebookClient(nb_man, **final_kwargs).execute()
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/papermill/clientwrap.py", line 45, in execute
    self.papermill_execute_cells()
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/papermill/clientwrap.py", line 72, in papermill_execute_cells
    self.execute_cell(cell, index)
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/data/aglinska/anaconda3/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/data/aglinska/anaconda3/lib/python3.8/site-packages/nbclient/client.py", line 820, in async_execute_cell
    raise DeadKernelError("Kernel died")
nbclient.exceptions.DeadKernelError: Kernel died
